opennms-config/src/main/java/org/opennms/netmgt/filter/JdbcFilterDao.java

```diff
@@ -83,6 +87,16 @@
 	private DataSource m_dataSource;
     private DatabaseSchemaConfig m_databaseSchemaConfigFactory;
 
+    private final static MetricRegistry metricRegistry = new MetricRegistry();
+    private final JmxReporter jmxReporter;
+    private final Timer getIpListTimer;
+
+    public JdbcFilterDao() {
+        getIpListTimer = metricRegistry.timer("getIPAddressList");
```

Comment:
Can we make the MBean domain/name more descriptive. Just by looking at the MBean, it's not clear what org.opennms.netmgt.config#getIPAddressList times.

Reply:
so ... org.opennms.netmgt.config.FilterDao#getIPAddressListForFilter ?
![image](https://user-images.githubusercontent.com/38085611/57161656-17135b80-6dba-11e9-803a-6d66cfc7ed53.png)

Reply:
That helps. I'd keep the FilterDao in the domain portion lower case though :octocat: